pkg/virt-handler/vm_test.go

```diff
@@ -1151,6 +1152,82 @@ var _ = Describe("VirtualMachineInstance", func() {
 			Expect(blockMigrate).To(BeTrue())
 			Expect(err).To(Equal(fmt.Errorf("cannot migrate VMI with non-shared HostDisk")))
 		})
+
+		Context("with network configuration", func() {
+			It("should block migration for bridge interface assigned to pod", func() {
+				vmi := v1.NewMinimalVMI("testvmi")
+				interface_name := "interface_name"
+
+				vmi.Spec.Networks = []v1.Network{
+					v1.Network{
+						Name:          interface_name,
+						NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}},
+					},
+				}
+				vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{
+					v1.Interface{
+						Name: interface_name,
+						InterfaceBindingMethod: v1.InterfaceBindingMethod{
+							Bridge: &v1.InterfaceBridge{},
+						},
+					},
+				}
+
+				err := controller.checkNetworkInterfacesForMigration(vmi)
+				Expect(err).To(HaveOccurred(), "should block migration for bridge binding")
+			})
+
+			It("should not block migration for bridge interface assigned to pod", func() {
+				vmi := v1.NewMinimalVMI("testvmi")
+				interface_name := "interface_name"
+
+				vmi.Spec.Networks = []v1.Network{
+					v1.Network{
+						Name:          interface_name,
+						NetworkSource: v1.NetworkSource{Pod: &v1.PodNetwork{}},
+					},
+				}
+				vmi.Spec.Domain.Devices.Interfaces = []v1.Interface{
+					v1.Interface{
+						Name: interface_name,
+						InterfaceBindingMethod: v1.InterfaceBindingMethod{
+							Masquerade: &v1.InterfaceMasquerade{},
+						},
+					},
+				}
+
+				err := controller.checkNetworkInterfacesForMigration(vmi)
+				Expect(err).ToNot(HaveOccurred(), "should not block migration for masquerade binding method")
+			})
+
+			It("should not block migration for bridge interface assigned to multus network", func() {
+				vmi := v1.NewMinimalVMI("testvmi")
+				interface_name := "interface_name"
```

Comment:
It will be nice to recognize that there are two interfaces, one primary and one secondary.
Something like this:
```
primaryInterface = "nic0"
secondaryInterface = "nic1"
```